### ISSUE TITLE
Auto-off timer is not canceled with a new print

### ISSUE THREAD

**Issue by CrNMGuy** [borisbu/OctoRelay#205]
When send a new print following a previous print, autooff timer is not canceled. Noticed through the new tootip "printer goes OFF in xx minutes [cancel]" while printing. When print end, autooff is reset.
Expected beahaviour : while printer is on an connected, if i send a new print, it should cancel autotimer, wait for print to end then init timer for autooff

Context : sending new print throught Prusa Slicer / Download and print  (connected to API)

**Comment by RobinTail** [comment]
@CrNMGuy ,

currently when start printing all previously postponed switches are cancelled, there are no exceptions.
Though, the new one can be set by the same event.

Since it's not clear what exactly and why is happening, please provide the debug level logs of the plugin (enable it in Logging section of OctoPrint) and a screenshot of your printer relay configuration.

**Comment by RobinTail** [comment]
Managed to reproduced on v3.9.0, @CrNMGuy .

```
2023-09-19 00:24:16,344 - octoprint.plugins.octorelay - DEBUG - Handling the plugin event PRINTING_STARTED having scope: ['r1', 'r2', 'r3', 'r4', 'r5', 'r6', 'r7', 'r8']
2023-09-19 00:24:16,573 - octoprint.plugins.octorelay - DEBUG - Cancelling tasks by request from PRINTING_STARTED for relay r1
2023-09-19 00:24:16,641 - octoprint.plugins.octorelay - DEBUG - The cancelled tasks removed from the registry
```

Probably the issue is happening because of the event setting `on Printing Started` and its default value `skip`.

![image](https://github.com/borisbu/OctoRelay/assets/13189514/a3b5228e-5edd-49cd-a083-6ae0dc5e9466)

**Comment by RobinTail** [comment]
Yeah, when the `target` is `None`, the postponed tasks are not cancelled.

https://github.com/borisbu/OctoRelay/blob/15818745ade5dc2fb6adf0c101394097ef461619/octoprint_octorelay/__init__.py#L191-L196

**Comment by CrNMGuy** [comment]
Sorry, didn't sent debug yet.
But you find the thing :)
Thanks !

**Comment by RobinTail** [comment]
🚀  The issue fixed in v3.9.1, @CrNMGuy 
Thank you for the contrubution!
